packages/store-cache/src/defaults.ts

```diff
@@ -0,0 +1,10 @@
+import { AbiType } from "@latticexyz/schema-type";
+
+// TODO: move to schema-type?
+// TODO: add default values for all abi types
+export const AbiDefaults = {
+  bytes: "0x00",
+  string: "",
+  uint256: 0n,
+  int32: 0,
+} as any satisfies { [key in AbiType]: unknown };
```

Comment:
does something like this exist in other packages? viem?

Reply:
not that I know of